src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -829,6 +849,7 @@ void MapDocumentActionHandler::updateActions()
                                                    [] (Layer *layer) { return layer->canMergeDown(); }));
     mActionSelectPreviousLayer->setEnabled(hasPreviousLayer);
     mActionSelectNextLayer->setEnabled(hasNextLayer);
+    mActionSelectAllLayers->setEnabled(true);
```

Comment:
Since the enabled state never changes, we can just leave out this line.

Reply:
Will do.